tests/integration/targets/azure_rm_backuppolicy/tasks/main.yml

```diff
@@ -0,0 +1,168 @@
+- name: Set Facts for Tests
+  set_fact:
+    vault_name: "rsv{{ resource_group | hash('md5') | truncate(22, True, '') }}"
+    location: "eastus"
+    policy_name_daily: "bp{{ resource_group | hash('md5') | truncate(22, True, '') }}"
```

Comment:
@coleneubauer policy_name_daily and policy_name_weekly names cannot be the same because the following deletions are made by name. Thank you very much!

Reply:
Good catch. I've now updated the names. Thanks